chalk-solve/src/solve/slg/aggregate.rs

```diff
@@ -69,6 +69,10 @@ impl<I: Interner> context::AggregateOps<SlgContext<I>> for SlgContextOps<'_, I>
                 break Guidance::Unknown;
             }
 
+            if let AnswerResult::Floundered = answers.peek_answer(|| should_continue()) {
+                break Guidance::Suggested(subst);
```

Comment:
Hmm, I think if we *floundered* we probably want `Guidance::Unknown`, as that suggests that we never really explored the options.